site/src/atomics/overflow.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Overflow
+description: Border related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Overflow
+
+At times, you'll need to determine the overflow behavior of an element. Atlas provides several classes to do this.
+
+| cssproperty                            | value              | screensize |
+| -------------------------------------- | ------------------ | ---------- |
+| `overflow`, `overflow-x`, `overflow-y` | `scroll`, `hidden` | N/A        |
```

Comment:
What about the `auto` value? I think in some cases it might be useful as well, for example, for tables.